lab-justin/gulpfile.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('default', function(){
+
+  return gulp.src('test/*.js')
+  .pipe(mocha());
+});
+
+gulp.task('eslint', function(){
+
+  return gulp.src('*.js')
```

Comment:
Make sure to ignore files and also make sure you're reading all the JS files in your project.  You'll want to change '*.js' to '**/*.js' and add '!node_modules' since you don't want to run your linter on node_modules.